api/controller.py

```diff
@@ -657,7 +657,7 @@ def groups(self, lane_identifier):
         feed = AcquisitionFeed.groups(
             self._db, title, url, lane, annotator, facets=facets
         )
-        return feed_response(feed.content)
+        return feed_response(feed)
```

Comment:
This uses my change from https://github.com/NYPL-Simplified/server_core/pull/874